<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>97</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="login_button">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>60</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
border-radius: 20px;
border: 3px solid white;
background-color: rgba(73, 150, 255, 158);</string>
       </property>
       <property name="text">
        <string>登录/注册</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mypage_button">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(73, 150, 255, 158);
border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
border-radius: 20px;
border: 3px solid white;</string>
       </property>
       <property name="text">
        <string>我的主页</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="search_button">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(73, 150, 255);
color: rgb(255, 255, 255);
border-radius: 20px;
border: 3px solid white;</string>
       </property>
       <property name="text">
        <string>钓书</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="adddemand_button">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(73, 150, 255, 158);
border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
border-radius: 20px;
border: 3px solid white;</string>
       </property>
       <property name="text">
        <string>捞书</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addbook_button">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(73, 150, 255, 158);
border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
border-radius: 20px;
border: 3px solid white;</string>
       </property>
       <property name="text">
        <string>放生</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="chat_button">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(73, 150, 255, 158);
border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
border-radius: 20px;
border: 3px solid white;</string>
       </property>
       <property name="text">
        <string>聊天墙</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="award_button">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(73, 150, 255, 158);
border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
border-radius: 20px;
border: 3px solid white;</string>
       </property>
       <property name="text">
        <string>成就墙</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>9</y>
      <width>681</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_3">
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>0</y>
           <width>531</width>
           <height>311</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="resource.qrc">:/image/logo.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QWidget" name="horizontalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>430</y>
           <width>281</width>
           <height>41</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="styleSheet">
             <string notr="true">border radium:20px
</string>
            </property>
            <property name="text">
             <string>注册</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="readytologin_button">
            <property name="styleSheet">
             <string notr="true">border radium:20px
</string>
            </property>
            <property name="text">
             <string>确认登录</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>320</y>
           <width>71</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;用户名：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>370</y>
           <width>71</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;密 码  ：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>320</y>
           <width>341</width>
           <height>81</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTextEdit" name="textEdit"/>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QTextEdit" name="textEdit_2"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="page">
        <widget class="QLineEdit" name="edit_searchline">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>10</y>
           <width>561</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 10px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>20</y>
           <width>141</width>
           <height>20</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(192, 192, 192);</string>
         </property>
         <property name="text">
          <string>请输入书名</string>
         </property>
        </widget>
        <widget class="QPushButton" name="ready_to_search">
         <property name="geometry">
          <rect>
           <x>610</x>
           <y>10</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-color: rgb(214, 214, 214);
</string>
         </property>
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>50</y>
           <width>58</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;选择tag:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_4">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>80</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 10px;
background-color: rgb(188, 229, 255);
border: 1px solid white;</string>
         </property>
         <property name="text">
          <string>数学</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_8">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>80</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 10px;
background-color: rgb(188, 229, 255);
border: 1px solid white;</string>
         </property>
         <property name="text">
          <string>计算机</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_9">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>80</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid white;
border-radius: 10px;
background-color: rgb(188, 229, 255);</string>
         </property>
         <property name="text">
          <string>英语</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_10">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>120</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid white;
border-radius: 10px;
background-color: rgb(188, 229, 255);</string>
         </property>
         <property name="text">
          <string>政治</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_11">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>120</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid white;
border-radius: 10px;
background-color: rgb(188, 229, 255);</string>
         </property>
         <property name="text">
          <string>法学</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_12">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>120</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid white;
border-radius: 10px;
background-color: rgb(188, 229, 255);</string>
         </property>
         <property name="text">
          <string>小说</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_13">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>80</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid white;
border-radius: 10px;
background-color: rgb(188, 229, 255);</string>
         </property>
         <property name="text">
          <string>心理</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_14">
         <property name="geometry">
          <rect>
           <x>490</x>
           <y>80</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid white;
border-radius: 10px;
background-color: rgb(188, 229, 255);</string>
         </property>
         <property name="text">
          <string>物理</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_15">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>80</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid white;
border-radius: 10px;
background-color: rgb(188, 229, 255);</string>
         </property>
         <property name="text">
          <string>化学</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_16">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>120</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid white;
border-radius: 10px;
background-color: rgb(188, 229, 255);</string>
         </property>
         <property name="text">
          <string>生物</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_17">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>120</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid white;
border-radius: 10px;
background-color: rgb(188, 229, 255);</string>
         </property>
         <property name="text">
          <string>历史</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_18">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>120</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid white;
border-radius: 10px;
background-color: rgb(188, 229, 255);</string>
         </property>
         <property name="text">
          <string>经管</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_19">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>80</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 10px;
background-color: rgb(188, 229, 255);
border: 1px solid white;</string>
         </property>
         <property name="text">
          <string>无</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_20">
         <property name="geometry">
          <rect>
           <x>490</x>
           <y>120</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid white;
border-radius: 10px;
background-color: rgb(188, 229, 255);</string>
         </property>
         <property name="text">
          <string>传记</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>220</y>
           <width>71</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;猜你喜欢：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>260</y>
           <width>121</width>
           <height>161</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="resource.qrc">:/image/math.jpeg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>260</y>
           <width>121</width>
           <height>161</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="resource.qrc">:/image/linear algebra.jpeg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_22">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>430</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Yuanti SC</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid rgb(188, 229, 255);
background-color: rgb(255, 255, 255);
border-radius: 10px;
</string>
         </property>
         <property name="text">
          <string>了解更多</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_23">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>430</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Yuanti SC</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid rgb(188, 229, 255);
background-color: rgb(255, 255, 255);
border-radius: 10px;
</string>
         </property>
         <property name="text">
          <string>了解更多</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2"/>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
